app/components/labs-ui/layer-group-toggle.js

```diff
@@ -39,6 +39,27 @@ export default Component.extend({
   actions: {
     toggle() {
       this.toggleProperty('active');
+      window.dataLayer = window.dataLayer || [];
+      if(this.get('label')==="Thematic Map") {
```

Comment:
Can you please fix the formatting here?  Spaces between if and parenthesis, between operator and operands, etc. Thank you

Reply:
I have included spaces between if and parenthesis.  Spacing between operator and operands I would like to remain, as I would like to maintain consistency with the code snippet Google provides.

Reply:
Wait which code snippet? Are you referring to https://github.com/NYCPlanning/labs-factfinder/pull/904/files#diff-12f0145db230b5cf853c879cf69ec9ffadbe7d85235f7b5db889ef5a627dae24R10-R17 

That's a script injection snippet and not meant to be human readable -- we shouldn't follow pattern for our code